components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -518,6 +518,10 @@ public static API getAPI(GovernanceArtifact artifact, Registry registry)
             api.setEndpointConfig(artifact.getAttribute(APIConstants.API_OVERVIEW_ENDPOINT_CONFIG));
 
             api.setRedirectURL(artifact.getAttribute(APIConstants.API_OVERVIEW_REDIRECT_URL));
+            api.setApiExternalProductionEndpoint(artifact.getAttribute(APIConstants
```

Comment:
Done from https://github.com/wso2/carbon-apimgt/pull/10979/commits/da4733c1f442eb70afb625f9f2ff51c5444a998d

Reply:
Had to add back properties for getAPIForPublishing method, since it is used
https://github.com/wso2/carbon-apimgt/pull/10979/commits/1135cf48a5892ac7257ae2cfad29afaeffea036e